integration-tests/src/test/resources/features/action-webhook.feature

```diff
@@ -25,7 +25,7 @@ Feature: Webhook Action tests
     And the Processor "myProcessor" of the Bridge "mybridge" has action of type "Webhook" and parameters:
       | endpoint | https://webhook.site/${env.webhook.site.uuid} |
 
-    When send a cloud event to the Ingress of the Bridge "mybridge" with path "events":
```

Comment:
@sutaakar just out of curiosity. What was the removed path refering to?

Was it some URI from https://github.com/cloudevents/sdk-java/blob/master/api/src/main/java/io/cloudevents/CloudEventAttributes.java ?

Reply:
@jomarko The missing path was added to Ingress endpoint URL, see https://issues.redhat.com/browse/MGDOBR-397
Therefore it is not needed to be added here any more.